spec.emu

```diff
@@ -605,8 +844,90 @@ contributors: Luca Casonato, Guy Bedford
       <emu-grammar><ins>ImportDeclaration : `import` `module` ImportedBinding FromClause `;`</ins></emu-grammar>
       <emu-alg>
         1. <ins>Let _localName_ be the sole element of BoundNames of |ImportedBinding|.</ins>
-        1. <ins>Let _reflectionEntry_ be the ImportEntry Record { [[ModuleRequest]]: _module_, [[ImportName]]: ~module~, [[LocalName]]: _localName_ }.</ins>
-        1. <ins>Return &laquo; _reflectionEntry_ &raquo;.</ins>
+        1. <ins>Let _entry_ be the ImportEntry Record { [[ModuleRequest]]: _module_, [[ImportName]]: ~module~, [[LocalName]]: _localName_ }.</ins>
+        1. <ins>Return &laquo; _entry_ &raquo;.</ins>
+      </emu-alg>
+    </emu-clause>
+  </emu-clause>
+</emu-clause>
+
+<emu-clause id="sec-fundamental-objects" number="20">
+  <h1>Fundamental Objects</h1>
+
+  <emu-clause id="sec-module-objects" number="6">
+    <h1><ins>Module Objects</ins></h1>
```

Comment:
We shouldn't need to define any of the `Module` object at this stage if we can null out the JS case.